### DISCUSSION TITLE
Thousands of Dispatched Events

### DISCUSSION THREAD

**Discussion by jtomek** [statamic/cms#5771] [Q&A] [answered]
Hello everybody,

Have you ever seen this? Some pages even get tens of thousands of these events. It's so slow! 

This screenshot relates to loading the control panel (no custom code involved).

I don't even know where to start the debugging process. There is only one dispatcher in the CMS located in the blueprint() method in the Auth\UserRepository.php. What calls the blueprint method??

<img width="943" alt="Screenshot 2022-04-06 at 16 23 34" src="https://user-images.githubusercontent.com/3748234/161997588-afe0f155-6f83-4a01-b19f-a7f4e31599a8.png">

**Comment by JohnathonKoster** [comment, marked as answer]
Hi there!

This is a known issue (https://github.com/statamic/cms/issues/5752), and I believe they are working on it 🙂

**Comment by jtomek** [comment]
Thank you for sharing this information with me.
